templates/zerver/api/get-user-by-email.md

```diff
@@ -0,0 +1,38 @@
+# Get a user by Email
```

Comment:
This, like other strings in Zulip, should be in sentence case (i.e. `email` should not be capitalized).